app/src/test/java/com/codesoom/assignment/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,43 @@
+package com.codesoom.assignment.utils;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.security.SignatureException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class JwtUtilTest {
+    private static final String SECRET = "12345678901234567890123456789010";
+    private static final String VALID_TOKEN =
+            "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
+
+    private static final String INVALID_TOKEN =
+            "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnTTS";
+    private JwtUtil jwtUtill;
+
+    @BeforeEach
+    void setUp(){
+        jwtUtill = new JwtUtil(SECRET);
+    }
+
+    @Test
+    void encode(){
+        String token = jwtUtill.encode(1L);
+        assertThat(token).isEqualTo(VALID_TOKEN);
+    }
+
+    @Test
+    void decodeWithValidToken(){
+        Claims claims = jwtUtill.decode(VALID_TOKEN);
+        assertThat(claims.get("userId", Long.class)).isEqualTo(1L);
+    }
+
+    @Test
+    void decodeWithInvalidToken(){
+        assertThatThrownBy(()->jwtUtill.decode(INVALID_TOKEN))
```

Comment:
여기도 이렇게 적절히 빈 칸을 넣어주는게 좋아보입니다. 에디터의 자동 정렬 기능을 사용해 보시는 것은 어떨까요? 심지어 저장될 때 마다 자동으로 되도록 할수도 있습니다
```suggestion
        assertThatThrownBy(() -> jwtUtill.decode(INVALID_TOKEN))
```